mantaray/backend.py

```diff
@@ -185,6 +185,8 @@ def __init__(self, server_config: config.ServerConfig, *, verbose: bool):
         self._ping_sent = False
         self._last_receive_time = time.monotonic()
 
+        self.nickmask: str | None = None
```

Comment:
- Please name this starting with `_` to prevent me from accidentally accessing it outside this class.
- This should be set to `None` when reconnecting. Compare it to `_ping_sent`, for example.